### PR TITLE
Eliminate all improper usages of the find_first utility function

### PR DESCRIPTION
**Pull request by kylegibson**
It's usage is problematic, since it encourages invalid assumptions about XML structure. Depends on #116

### CODE REVIEW

**Comment by jlward** [comment]
Can you merge this up with master to make reviewing easier?

File: pydocx/util/xml.py [hunk 1]
```diff
@@ -40,6 +40,8 @@ def el_iter(el):
 def find_first(el, tag):
     """
     Find the first occurrence of a tag beneath the current element.
+
+    This method should not be used to navigate an OOXML schema
```
Comment (jlward) [inline pydocx/util/xml.py]: How do you feel about adding a `DepricationWarning` here?

**Comment by jlward** [comment]
This looks fine.

File: pydocx/util/xml.py (hunk 1 above)
Comment (kylegibson) [inline pydocx/util/xml.py]: We could do that... although technically since we're < 1.0 it's not required. In alpha, everything is pending deprecation.

Comment (jlward) [inline pydocx/util/xml.py]: Lol, fair enough.

**Pull request merged**